packages/sui-ssr/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@s-ui/ssr",
-  "version": "5.10.0",
+  "version": "5.10.0-beta.4",
```

Comment:
did you forget this?

Reply:
This will be overridden with the release